cpp/oneapi/dal/algo/svm/common.cpp

```diff
@@ -130,12 +124,39 @@ void descriptor_base<Task>::set_kernel_impl(const detail::kernel_function_ptr& k
     impl_->kernel = kernel;
 }
 
+template <typename Task>
+void descriptor_base<Task>::set_class_count_impl(std::int64_t value) {
+    if (value <= 1) {
+        throw domain_error(dal::detail::error_messages::class_count_leq_one());
+    }
+    impl_->class_count = value;
+}
+
+template <typename Task>
+std::int64_t descriptor_base<Task>::get_class_count_impl() const {
+    return impl_->class_count;
+}
+
+template <typename Task>
+void descriptor_base<Task>::set_epsilon_impl(double value) {
+    if (value < 0.0) {
+        throw domain_error(dal::detail::error_messages::epsilon_leq_zero());
```

Comment:
Inconsistency between condition and error message name

Reply:
Not addressed. You have 'less than' in the condition and 'lest than or equal to' in error message name

Reply:
applied